en/assembly/mount_and_orient_controller.md

```diff
@@ -13,6 +18,14 @@ The controller should be placed on the frame top-side up, oriented so that the a
 If the controller cannot be mounted in the recommended/default orientation (e.g. due to physical constraints) you will need to configure the autopilot software with the orientation that you actually used: [Flight Controller Orientation](../config/flight_controller_orientation.md).
 :::
 
+## Position
+
+The flight controller should be placed on the frame as close as possible to the centre-of-gravity.
+
+If you can't mount the controller in this position, then you can [configure](../advanced_config/parameters.md) the following parameters to set offset relative to the CoG: [EKF2_IMU_POS_X](../advanced_config/parameter_reference.md#EKF2_IMU_POS_X), [EKF2_IMU_POS_Y](../advanced_config/parameter_reference.md#EKF2_IMU_POS_Y), [EKF2_IMU_POS_Z](../advanced_config/parameter_reference.md#EKF2_IMU_POS_Z) (for the default EKF2 estimator).
+
+Note that if you don't set these offsets then EKF2 position/velocity estimates will be at the IMU location rather that at the CoG.
```

Comment:
@bresch What would you "see" in this case - i.e how big a difference does it make? Are there conditions in which it would make a bigger difference?

Reply:
You'll see the difference is position control mode for example. If the IMU is in front of the c.g., a pitching motion (around the c.g.) will result in a reduction of the altitude estimate (which is correct, the IMU goes down); as a reaction, the altitude controller will give more thrust to compensate for it. The amplitude depends on how far the IMU is located from the c.g. and can be negligible in some cases but it's still some unneeded control effort that is constantly applied.

If the offsets are specified, a pure pitch motion would not create any change in the altitude estimate so there will be less parasitic corrections.